bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableDataGrpcClient.java

```diff
@@ -267,18 +242,40 @@ public void run() {
       }
     }, executorService);
 
+    int bufferCapacity = retryOptions.getStreamingBufferSize();
```

Comment:
nit: move this down to clientCallService.asyncServerStreamingCall() where it is used.